### DISCUSSION TITLE
how to recover from this

### DISCUSSION THREAD

**Discussion by jos705** [hapostgres/pg_auto_failover#852] [Q&A] [answered]
```
[postgres@j1lpg001 log]$ pg_autoctl show state
  Name |  Node |     Host:Port |       TLI: LSN |   Connection |      Reported State |      Assigned State
-------+-------+---------------+----------------+--------------+---------------------+--------------------
node_1 |     1 | j1lpg001:5432 |  12: 0/D19EF18 |       none ! |             demoted |          catchingup
node_3 |     3 | j1lpg003:5432 |  13: 0/D19F068 |   read-write |        wait_primary |        wait_primary
```

got this state after I did a pg_autoctl stop on node_1.  But can't recover from it

I'm testing out pg_auto_failover this week, i'm pretty new to postgres and HA. So any tips would me much appreciated.

**Comment by redbaron** [comment, marked as answer]
`wait_primary` means the node still has no failover, but there’s hope for a secondary to synchronize with it soon. You need to bring up secondary and make it catchup with node_3 for node_3 to become `primary`. In the current state read-write requests can go to node_3, but you have no redundancy should `node_3` fail

**Reply by jos705** [reply]
I got it, thanks
The reason the secondairy node wasn't coming up anymore was because I added a line to my pg_hba.comf which made a connection to node_3 need a password. I saw this in the output: 
INFO  pg_rewind: fatal: fe_sendauth: no password supplied
After I fixed the pg_auto.conf on node 3 pg_autoctl  recoverd node_1. Nice!!
